documents.tf

```
@@ -19,4 +19,13 @@ module "ticf-cri-infrastructure" {
   document_path = "fraud/ticf-cri/ticf-cri-infrastructure.json"
   document_name = "TICF-CRI Infrastructure Dashboard"
   document_type = "dashboard"
+}
+
+### Data Pod ###
+
+module "signal-exchange-production-dashboard" {
+  source        = "./modules/documents"
+  document_path = "signal-exchange/signal-exchange-production-dashboard"
```

The TF plan actions for this are failing atm, hopefully just a case of this: 
```suggestion
  document_path = "signal-exchange/signal-exchange-production-dashboard.json"
```